build_scripts/docker/docker.sh

```diff
@@ -37,17 +38,10 @@ copy_server_config_to_build_dir() {
 
 build_package() {
   local version=$1
-  local commit_id=$2
-  local dist_dir=$3
+  local dist_dir=$2
   pushd ./docker
 
-  if [ -n "$1" ]; then
-    version="v$version"
-  else
-    version="$commit_id"
-  fi
-
-  docker_image_name="guardicore/monkey-island:$version"
+  docker_image_name="$DOCKER_IMAGE_NAME:$version"
```

Comment:
I don't love having `DOCKER_IMAGE_NAME` and `docker_image_name`. I can't find a term for a name and a tag together, so I'm not really sure what else to call it.

What about removing this variable and just building the string inline? It only gets used in one place.

`build_docker_image_tar "$DOCKER_IMAGE_NAME:$version" "$tar_name"`

Reply:
I didn't like it either but I left it for readability. I will do it inline.